build/specifications/DotNet.json

```diff
@@ -125,6 +125,12 @@
             "itemFormat": "{key}={value}",
             "disallowedCharacter": ";",
             "help": "<p>Set or override the specified project-level properties, where name is the property name and value is the property value. Specify each property separately, or use a semicolon or comma to separate multiple properties, as the following example shows:</p><p><c>/property:WarningLevel=2;OutDir=bin\\Debug</c></p>"
+          },
+          {
+            "name": "BlameMode",
+            "type": "bool",
+            "format": "--blame",
+            "help": "Runs the tests in blame mode. This option is helpful in isolating the problematic tests causing test host to crash. It creates an output file in the current directory as *Sequence.xml* that captures the order of tests execution before the crash."
```

Comment:
This doesn’t support markdown. So `<emph>`.

Reply:
updated